src/script/squirrel_helper.hpp

```diff
@@ -15,6 +15,7 @@
 #include "../economy_type.h"
 #include "../string_func.h"
 #include "squirrel_helper_type.hpp"
+#include <3rdparty/squirrel/include/squirrel.h>
```

Comment:
Regardless, should be `#include <squirrel.h>` (3rdparty/squirrel/include is on the searchpath)